### DISCUSSION TITLE
How can I find the web3 platform Patrick mentioned?

### DISCUSSION THREAD

**Discussion by thurendous** [Cyfrin/foundry-full-course-cu#188] [Q&A] [answered]
I remember that Patrick mentioned a web3-ish platform for asking questions and help each other like Stack overflow. I forgot the name and cannot find the link to it. 

Could anyone help me about this? Give me a link please.

**Comment by 0xgun** [comment]
It is [Ethereum Stack Exchange](https://ethereum.stackexchange.com/)

**Reply by thurendous** [reply]
Thank you too. I was finding this: https://peeranha.io/feed

**Comment by palmcivet7** [comment, marked as answer]
He mentioned [Peeranha](https://peeranha.io/feed) too. There's a list of resources for the course [here in the README](https://github.com/Cyfrin/foundry-full-course-f23#resources-for-this-course).

**Reply by thurendous** [reply]
Thank you sir! You made my day!
